token-repository/src/test/scala/io/renku/tokenrepository/repository/creation/TokensCreatorSpec.scala

```diff
@@ -244,22 +243,27 @@ class TokensCreatorSpec extends AnyWordSpec with MockFactory with should.Matcher
 
       givenSuccessfulTokensRevoking(projectId, tokenCreationInfo, userAccessToken)
 
-      tokensCreator.create(projectId, userAccessToken) shouldBe ().pure[Try]
+      tokensCreator.create(projectId, userAccessToken).unsafeRunSync() shouldBe ()
     }
 
     "fail if finding stored token fails" in new TestCase {
 
-      val exception = exceptions.generateOne.raiseError[Try, EncryptedAccessToken]
-      givenStoredTokenFinder(projectId, returning = OptionT.liftF(exception))
+      val exception = exceptions.generateOne
+      givenStoredTokenFinder(projectId, returning = OptionT.liftF(exception.raiseError[IO, EncryptedAccessToken]))
+
+      val ex =
+        intercept[Exception] {
+          tokensCreator.create(projectId, userAccessToken).unsafeRunSync()
+        }
 
-      tokensCreator.create(projectId, userAccessToken) shouldBe exception
+      ex shouldBe exception
     }
 
-    "fail if retry process hit the max number of attempts" in new TestCase {
+    "fail if reIO process hit the max number of attempts" in new TestCase {
```

Comment:
one more

Reply:
Oh no, thank you! That search&replace went a bit too far :)